File: Code/HVIT/HVIT_EX/HVIT_OOP_TEST/OOP/SanPham/SanPham.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SanPham
{
    class SanPham
    {
        public int MaSanPham { get; set; }
        public string TenSanPham { get; set; }
        public string LoaiSanPham { get; set; }
        public bool LaSanPhamMoi { get; set; }
        public SanPham(int ma, string ten, string loaiSanPham, bool laSPMoi)
            => (MaSanPham, TenSanPham, LoaiSanPham, LaSanPhamMoi) = (ma, ten, loaiSanPham, laSPMoi);
        public void HienThi()
            => Console.WriteLine($"{MaSanPham} san pham {TenSanPham} thuoc loai {LoaiSanPham} {LaSanPhamMoi} la san pham moi");
    }
}

File: Code/HVIT/HVIT_JAVA/src/JavaPractice/JavaOOPBasic/SanPham/SanPham.java
package JavaPractice.JavaOOPBasic.SanPham;

import java.util.Scanner;

public class SanPham {
    private int MaSanPham;
    private String TenSanPham;
    private String LoaiSanPham;
    private Boolean LaSanPhamMoi;

    public int getMaSanPham() {
        return MaSanPham;
    }

    public void setMaSanPham(int maSanPham) {
        MaSanPham = maSanPham;
    }

    public String getTenSanPham() {
        return TenSanPham;
    }

    public void setTenSanPham(String tenSanPham) {
        TenSanPham = tenSanPham;
    }

    public String getLoaiSanPham() {
        return LoaiSanPham;
    }

    public void setLoaiSanPham(String loaiSanPham) {
        LoaiSanPham = loaiSanPham;
    }

    public Boolean getLaSanPhamMoi() {
        return LaSanPhamMoi;
    }

    public void setLaSanPhamMoi(Boolean laSanPhamMoi) {
        LaSanPhamMoi = laSanPhamMoi;
    }

    public SanPham(int maSP, String tenSP, String loaiSP, boolean laSanPhamMoi) {
        setMaSanPham(maSP);
        setTenSanPham(tenSP);
        setLoaiSanPham(loaiSP);
        setLaSanPhamMoi(laSanPhamMoi);
    }

    public void InThongTin() {
        System.out.println("San pham " + getTenSanPham() + " co ma la " + getMaSanPham() + " thuoc loai san pham " + getLoaiSanPham() + " la san pham moi " + getLaSanPhamMoi());
    }
}
